### ISSUE TITLE
Highlighting code-tools source code not legible with highlight-style: a11y-dark 

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [quarto-dev/quarto-cli#1370]
### Bug description

Adding the following to the YAML of the template Quarto Document results in source code that is too light to be legible.

``` yaml
format: 
  html:
    code-tools: true
    highlight-style: a11y-dark
```

<img width="1552" alt="Screen Shot 2022-07-12 at 6 59 10 PM" src="https://user-images.githubusercontent.com/5965649/178654546-99f28b1b-f9b9-4ee2-b97d-c59f81e652f7.png">

- Quarto version: 1.0.16
- OS: macOS 12.4
- RStudio: RStudio 2022.07.0+548 "Spotted Wakerobin" Release (34ea3031089fa4e38738a9256d6fa6d70629c822, 2022-07-06) for macOS Mozilla/5.0 (Macintosh; Intel Mac OS X 12_4_0) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.10 Chrome/69.0.3497.128 Safari/537.36

### `quarto check` Output

```
[✓] Checking Quarto installation......OK
      Version: 1.0.16
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.1
      Path: /usr/local/bin/python3
      Jupyter: 4.9.1
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.2.0
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Users/mine/Desktop/RStudio/quarto-tip-a-day/renv/library/R-4.2/x86_64-apple-darwin17.0
        - /Library/Frameworks/R.framework/Versions/4.2/Resources/library
      rmarkdown: 2.14

[✓] Checking Knitr engine render......OK
```

### `quarto tools check` Output

```
Tool         Status                    Installed     Latest  
chromium     Not installed             ---           869685  
tinytex      External Installation     ---           v2022.07
```

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [x] documented the quarto version you're running, by pasting the output from running `quarto check` in the "Quarto Check Output" text area?
- [x] #1374
- [X] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by dragonstyle** [comment]
This is by design, currently.  `a11y` is one of our adaptive text-highlighters, which will automatically adapt to the code block background color and select the appropriate dark or light mode. As a result, it doesn't carry a background color directly, instead using the code background color to select between the light and dark modes. I would recommend that you instead just select the highlight theme `a11y` and allow it to adapt, or use `a11y-light` instead (which is designed to work against light backgrounds), or if you'd like to use the dark mode explicitly, color the code blocks to provide more contrast.

Does this make sense?

**Comment by dragonstyle** [comment]
Ohhhhh - now I see that the issue is code tools code display itself. Let me take a closer look at this!

**Comment by cderv** [comment]
@mine-cetinkaya-rundel should be fixed in v1.0.18 build.
